### PR TITLE
Added support for php 8.2

### PR DESCRIPTION
**Pull request by o-perator**
Added support for php 8.2

The project code has been brought back to normal in accordance with the new checks in cs-fixer

### CODE REVIEW

**Comment by jonwayne-tithely** [comment]
Other than renaming the project in composer, these changes look like something that is needed. Is there a reason for closing it?

**Comment by o-perator** [comment]
Hi)
When opening PR, I missed it, and instead of my fork I opened it to the main repository)
A little later I will issue a normal PR)

File: .gitignore [hunk 1]
```diff
@@ -2,3 +2,4 @@
 build/
 composer.lock
 vendor/
+.idea
```
Comment (1ma) [inline .gitignore:5]: Remove IDE-specific .gitignore directive.

**Review by 1ma — CHANGES_REQUESTED** [review]

**Comment by 1ma** [comment]
LGTM except for that little thing! Thanks for your contribution.

File: .gitignore (hunk 1 above)
Comment (o-perator) [reply .gitignore]: removed

**Review by 1ma — APPROVED** [review]

**Comment by 1ma** [comment]
@o-perator @jonwayne-tithely tagged as 4.1.0

**Pull request merged**